x/emissions/keeper/keeper.go

```diff
@@ -2011,6 +2008,10 @@ func (k *Keeper) GetTopic(ctx context.Context, topicId TopicId) (types.Topic, er
 
 // Sets a topic config on a topicId
 func (k *Keeper) SetTopic(ctx context.Context, topicId TopicId, topic types.Topic) error {
+	_, err := k.IncrementTopicId(ctx)
```

Comment:
uhhhhh i think this is going to break a lot of unit tests that expect to be able to set topics

Reply:
Just reverted this change